src/compiler/scala/tools/nsc/settings/ScalaSettings.scala

```diff
@@ -283,6 +283,7 @@ trait ScalaSettings extends StandardScalaSettings with Warnings { _: MutableSett
   val YpickleWrite = StringSetting("-Ypickle-write", "directory|jar", "destination for generated .sig files containing type signatures.", "", None).internalOnly()
   val YpickleWriteApiOnly = BooleanSetting("-Ypickle-write-api-only", "Exclude private members (other than those material to subclass compilation, such as private trait vals) from generated .sig files containing type signatures.").internalOnly()
   val YtrackDependencies = BooleanSetting("-Ytrack-dependencies", "Record references to in unit.depends. Deprecated feature that supports SBT 0.13 with incOptions.withNameHashing(false) only.", default = true)
+  val Yscala3ImplicitResolution = BooleanSetting("-Yscala3-implicit-resolution", "Use Scala-3-style downwards comparisons for implicit search and overloading resolution", default = false)
```

Comment:
Maybe add a link to https://github.com/scala/bug/issues/12437 in the description

Reply:
Will do 👍

Reply:
I've made a change like this: https://github.com/scala/scala/pull/10012/files#diff-87059578fba24c0fb0b0fcca877242dff060ac85c89dbf2dfd04548d477f8e6bR278